packages/atomic/src/components.d.ts

```diff
@@ -186,7 +186,7 @@ export namespace Components {
          */
         "injectionDepth": number;
         /**
-          * Specifies if the facet is collapsed.
+          * Specifies if the facet is collapsed. When the facet is the child of an `atomic-facet-manager` component, it will control this property.
```

Comment:
Here, by `it`, do you mean the `atomic-facet-manager`? If so, I suggest:
```suggestion
          * Specifies whether the facet is collapsed. When the facet is the child of an `atomic-facet-manager` component, the facet manager controls this property.
```

Reply:
This sounds better to me too :P

Reply:
Thanks!